src/Form/inputTypeConfig.js

```diff
@@ -78,12 +78,13 @@ const canBeInteractedWith = (input) => !input.readOnly && !input.disabled
  * @returns {Promise<boolean>}
  */
 const canBeAutofilled = async (input, device) => {
-    if (!canBeInteractedWith(input)) return false
```

Comment:
This fixes Lufthansa. Basically, we don't want to decide whether we can autofill an input based on whether it can be interacted with. We use interactivity to decide whether to show the icon, but that's done elsewhere.